File: library/src/main/java/me/proxer/library/entitiy/ProxerImageItem.java
package me.proxer.library.entitiy;

/**
 * An interface for all entities with an image.
 *
 * @author Ruben Gees
 */
public interface ProxerImageItem {

    /**
     * Returns the image.
     * <p>
     * This can either be an id or a direct link, depending on the API.
     */
    String getImage();
}

File: library/src/main/kotlin/me/proxer/library/entity/ProxerImageItem.kt
package me.proxer.library.entity

/**
 * An interface for all entities with an image.
 *
 * @author Ruben Gees
 */
interface ProxerImageItem {

    /**
     * The image.
     * This can either be an id or a direct link, depending on the API.
     */
    val image: String
}
